Mini/MiniCooperSE.json

```
@@ -0,0 +1,71 @@
+{
+  "init_commands": {
+    "command": [
+	"ATZ", 
+	"ATD", 
+	"ATE0", 
+	"ATS0", 
+	"ATAL",
+	"ATPBE101",
+	"ATSPB",
+	"ATBI",
+	"ATSH6F1",
+	"ATAT0",
+	"ATSTFF",
+	"ATCRA607",
+        "ATCEA07",
+        "ATFCSH6F1",
+        "ATFCSD07300800",
+        "ATFCSM1"
+	]
+  },
+  "data_commands": {
+    "command": [
+      "22DDBC",
+      "22DD68",
+      "22DD69",
+      "226335"
+    ]
+  },
+  "obd_protocol": "B",
```

```suggestion
  "obd_protocol": "B",
```
We don't have protocol B, so we only have 6 and 7 right now, will check if one of those works with the data.

---

maybe 6 or 7 will work as well, i read a comment in another pull request that the ATSP7 command demands protocol 7. But in the init sequence it is ATSPB (set protocol B)

---

Protocfol 6 here works.